conda-store-server/conda_store_server/server/views/ui.py

```diff
@@ -2,6 +2,9 @@
 
 import yaml
 from conda_store_server import api
+from conda_store_server.action.generate_constructor_installer import (
+    get_installer_platform,
+)
```

Comment:
The UI now prints the architecture/platform that's actually built by constructor.